src/main/java/com/Futurecollars/Lekcja7/Zadanie2/BigLetters.java

```diff
@@ -0,0 +1,17 @@
+package com.Futurecollars.Lekcja7.Zadanie2;
+
+public class BigLetters {
+
+    public static String toUpperCase(String s){
+        String str = "";
+        for(int x=0; x<s.length(); x++){
```

Comment:
jak s będzie nullem to poleci exception. Testy które masz najprawdopodobniej ci nie przechodzą, sprawdź to